pinot-query-planner/src/test/resources/queries/JoinPlans.json

```diff
@@ -294,19 +294,21 @@
           "\n                                LogicalFilter(condition=[=($0, _UTF-8'foo')])",
           "\n                                  LogicalTableScan(table=[[default, b]])",
           "\n              PinotLogicalExchange(distribution=[hash[0]])",
-          "\n                PinotLogicalAggregate(group=[{0}], agg#0=[MIN($1)])",
-          "\n                  PinotLogicalExchange(distribution=[hash[0]])",
-          "\n                    PinotLogicalAggregate(group=[{0}], agg#0=[MIN($1)])",
-          "\n                      LogicalProject(col3=[$2], $f1=[true])",
-          "\n                        LogicalFilter(condition=[=($0, _UTF-8'bar')])",
-          "\n                          LogicalTableScan(table=[[default, b]])",
+          "\n                LogicalProject(col3=[$0], $f1=[$1])",
```

Comment:
Interesting, I think this project is redundant? Something to note regardless as we work on a proper long term solution. Same in the other parts of this plan.

Reply:
Agreed, we can address this subsequently.